pmd-apex/pom.xml

```diff
@@ -105,7 +105,7 @@
           <plugin>
```

Comment:
I think, we should check, whether this profile actually still works and whether this is documented.
The goal was, if I remember correctly, to easily start the designer from the current workspace like

```
cd pmd-apex
./mvnw -Pdesigner org.codehaus.mojo:exec-maven-plugin:java
# or ./mvnw -Pdesigner org.codehaus.mojo:exec-maven-plugin:exec ??
```

I'm not sure, whether this still works, since the designer is maybe not on the classpath? And it is anyway in a different repository (https://github.com/pmd/pmd-designer)... 

To start the designer, one should follow the instructions on https://github.com/pmd/pmd/wiki/Setup-IDE

I'd suggest to remove this profile completely

Reply:
That'll require some time to investigate. I'll check whether this works. Is it correct that pmd-designer was once in this pmd repository?

Reply:
I think, it's a good idea to remove this. Probably the designer pom could have a runnable profile, but it's kind of random to put one in the apex module.

Reply:
> Is it correct that pmd-designer was once in this pmd repository?

Yes, it was once part of the main repo (see https://github.com/pmd/pmd/tree/master/pmd-ui) but is now a separate project. The designer is pulled in via a dependency (in pmd-dist: https://github.com/pmd/pmd/blob/534bfe3c55c73a55065815598ee82e9260736054/pmd-dist/pom.xml#L262-L266)

Reply:
```
cd pmd-apex
../mvnw -Pdesigner org.codehaus.mojo:exec-maven-plugin:java
```

actually works - it starts the old, deprecated, designer, which we shouldn't promote.
I'll remove this profile.